extensions/amp-accordion/amp-accordion.md

```diff
@@ -70,6 +71,10 @@ Example:
 
 ## Attributes
 
+**disable-session-states**
+
+The `disable-session-states` attribute can be set on `<amp-accordion>` to opt out of preserving state of `amp-accordion` element.
```

Comment:
w/ @bpaduch 's edits this becomes: "The `disable-session-states` attribute can be set on `<amp-accordion>` to opt out of preserving the state of the `amp-accordion` element."

I would add the clarification that this refers to the expanded/collapsed state. I suggest:

"The `disable-session-states` attribute can be set on `<amp-accordion>` to opt out of preserving the collapsed/expanded state of the `amp-accordion` element."